src/Transport/BirdMailTransport.php

```diff
@@ -166,7 +166,7 @@ private function getFrom(Email $email): ?array
 
         return [
             'displayName' => $from->getName(),
-            'username' => $from->getAddress(),
+            'username' => collect(explode('@', $from->getAddress()))->first(),
```

Comment:
What about using: https://laravel.com/docs/9.x/helpers#method-fluent-str-before?

Reply:
Not a bad idea. I'll change it to that.